targets/targets.json

```diff
@@ -534,10 +534,21 @@
         "is_disk_virtual": true,
         "inherits": ["Target"],
         "detect_code": ["0218"],
-        "device_has": ["ANALOGIN", "ANALOGOUT", "ERROR_RED", "I2C", "I2CSLAVE", "INTERRUPTIN", "PORTIN", "PORTINOUT", "PORTOUT", "PWMOUT", "RTC", "SEMIHOST", "SERIAL", "SLEEP", "SPI", "SPISLAVE", "STDIO_MESSAGES"],
```

Comment:
Thanks ! Be careful though KL82Z is actually a board not MCU (at least that is what I noticed in targets.json), thus there should be MCU KL82Z and MCU K82F ?